experimental/examples/bubblebench/editable-shared-tree/src/Client.ts

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IClient } from "@fluid-example/bubblebench-common";
+import { brand, EditableField, FieldKey, JsonableTree, singleTextCursor } from "@fluid-internal/tree";
+import { Bubble } from "./Bubble";
+import { ClientTreeProxy, iBubbleSchema, numberSchema } from "./schema";
+
+export class Client implements IClient {
+    static bubblesFieldKey: FieldKey = brand("bubbles");
+
+    constructor(public readonly clientTreeProxy: ClientTreeProxy) {}
+
+    public get clientId() {
+        return this.clientTreeProxy.clientId;
+    }
+
+    public set clientId(value: string) {
+        this.clientTreeProxy.clientId = value;
+    }
+
+    public get color() {
+        return this.clientTreeProxy.color;
+    }
+    public set color(value: string) {
+        this.clientTreeProxy.color = value;
+    }
+
+    public get bubbles() {
+        return [...this.clientTreeProxy.bubbles].map(
+            (bubbleTreeProxy) => new Bubble(bubbleTreeProxy),
+        );
+    }
+
+    public increaseBubbles(bubble: { x: number; y: number; r: number; vx: number; vy: number; }) {
+        const newBubbleJson: JsonableTree = {
+            type: iBubbleSchema.name,
+            fields: {
+                x: [{ type: numberSchema.name, value: bubble.x }],
+                y: [{ type: numberSchema.name, value: bubble.y }],
+                r: [{ type: numberSchema.name, value: bubble.r }],
+                vx: [{ type: numberSchema.name, value: bubble.vx }],
+                vy: [{ type: numberSchema.name, value: bubble.vy }],
+            },
+        };
+
+        const bubblesSequenceNode = this.clientTreeProxy[Client.bubblesFieldKey] as EditableField;
+        bubblesSequenceNode.insertNodes(
+            bubblesSequenceNode.length,
+            singleTextCursor(newBubbleJson),
+        );
```

Comment:
If you make the schema change I suggested, you can update this to:

```suggestion
        const bubblesSequenceNode: EditableField = this.clientTreeProxy.bubbles;
        bubblesSequenceNode[bubblesSequenceNode.length] = bubble;
```

Reply:
Done in most recent commit.

Reply:
It seems like this change was not done (or was reverted).